lib/content/audio/passthrough.ex

```diff
@@ -41,7 +41,11 @@ defmodule Content.Audio.Passthrough do
     end
 
     defp tts_text(%Content.Audio.Passthrough{} = audio) do
-      train = PaEss.Utilities.train_description(audio.destination, audio.route_id)
+      train =
+        if audio.destination in [:ashmont, :braintree],
+          do: PaEss.Utilities.train_description(:southbound, audio.route_id),
+          else: PaEss.Utilities.train_description(audio.destination, audio.route_id)
+
```

Comment:
There's a bit of logic in `Signs.Utilities.Predictions.prediction_passthrough_audios` that does some munging of the destination, which should be updated. Actually, that might be a more natural place to override the destination to `:southbound`, and then this module could directly map that.